requirements.adoc

```diff
@@ -49,20 +53,16 @@ For the hub cluster networking requirements, see the following table:
 | Kubernetes API server of the provisioned managed cluster
 | 6443
 
-| Outbound
-| The channel source, including GitHub, Object Store, and Helm repository. This is only required when you are using Application lifecycle to connect to these sources.
-| 
-
 | Outbound and inbound
 | The `WorkManager` service route on the managed cluster
 | 443
 
 | Inbound
-| The kube API server of the hub cluster from the managed cluster
+| The kube API server of the multicluster engine for Kubernetes cluster from the managed cluster
```

Comment:
Using "kube" her seems like using slang.  I think this should start "The Kubernetes API server..."